wd_helper.au3

```diff
@@ -636,6 +636,199 @@ Func _WD_FrameLeave($sSession)
 	Return SetError(__WD_Error($sFuncName, $_WD_ERROR_Success), 0, $sValue)
 EndFunc   ;==>_WD_FrameLeave
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_FrameList
```

Comment:
@Danp2 please reaview `header` and in code `comments` for `_WD_FrameList`

Reply:
This will get done in due time